kloss.md

```diff
@@ -0,0 +1,7 @@
+Eric Kloss
+
+Kloss was born blind in Greenville, Pennsylvania, near Pittsburgh, and attended the Western Pennsylvania School for the Blind, which was run by his father. When he was 10, he started on saxophone, and two years later he was playing in night clubs with professional musicians such as Bobby Negri, Charles Bell, and Sonny Stitt. At 16, he recorded his debut album, Introducing Eric Kloss (Prestige, 1965) with Don Patterson and Pat Martino.[1]
+
+On his third album, Grits & Gravy (1966), he was recording with musicians over twice his age: Jaki Byard, Richard Davis, and Alan Dawson. He continued recording and performing while a student at Duquesne University. A fan of Elvis Presley and The Ventures, he was attracted to the growth of jazz fusion in the 1960s and '70s, and eventually played with fusion musicians Chick Corea, Dave Holland, and Jack DeJohnette.[1] He also collaborated with Richie Cole and Gil Goldstein, and did sessions with Cedar Walton, Jimmy Owens, Kenny Barron, Jack DeJohnette, Booker Ervin, Chick Corea, Barry Miles, and Terry Silverlight.[2]
+
+In the 1980s, Kloss taught at Rutgers University, then Duquesne and Carnegie Mellon. He and his wife, a vocalist, collaborated in a group called Quiet Fire. He has performed and recorded rarely since the 1980s due to health problems.[1]
```

Comment:
```suggestion
In the 1980s, Kloss taught at Rutgers University, then Duquesne and Carnegie Mellon. He lived in New Jersey for a while. He and his wife, a vocalist, collaborated in a group called Quiet Fire. He has performed and recorded rarely since the 1980s due to health problems.[1]
```